spot_wrapper/spot_arm.py

```diff
@@ -447,9 +447,10 @@ def hand_pose(
         qy: float,
         qz: float,
         qw: float,
-        duration: float = 0.125,
+        duration: float = 0.0,
```

Comment:
Will this cause an incompatibility with other users who may be relying on the nonzero default arg?

Reply:
Seeing as I am the only user right now.... probably not?